core/src/components/badge/badge.ionic.scss

```diff
@@ -0,0 +1,29 @@
+@use "../../foundations/ionic.vars.scss" as tokens;
```

Comment:
We should instead change this to `@use "../../themes/ionic/ionic.globals.scss" as globals;`. This new approach was suggested after the creation of the ionic.globals.scss partial, that already forwards all the necessary tokens for Ionic Theme

Reply:
This si detailed on the foundations ReadME: https://github.com/ionic-team/ionic-framework/blob/next/core/src/foundations/README.md#how-to-use